### PR TITLE
share security context between api and frontend, update somposer.json

### PR DESCRIPTION
**Pull request by ahilles107**
remember_me is not needed for oauth - time is controlled by token.
when user logs into oauth_authorize (to get api access_token) then we login him also to frontend. 
use tagged knplabs/knp-menu release instead -dev

### CODE REVIEW

File: newscoop/composer.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
         "hybridauth\/hybridauth":"dev-master",
         "smarty\/smarty":"3.1.21",
         "knplabs\/knp-menu-bundle":"2.0.*@dev",
-        "knplabs\/knp-menu":"2.0.*@dev",
+        "knplabs\/knp-menu":"2.0.1",
```
Comment (m038) [inline newscoop/composer.json]: :+1: Good catch

Comment (ahilles107) [inline newscoop/composer.json]: i will update it later - so far i have some plugins installed and don't want to crash anything

Comment (ahilles107) [inline newscoop/composer.json]: thinking also about fixing other libraries to tagged versions

Comment (m038) [inline newscoop/composer.json]: :100: that would be great, we've had some issues with dev versions before.

**Pull request merged**